pyccl/neutrinos.py

```diff
@@ -75,7 +75,8 @@ def nu_masses(OmNuh2, mass_split, T_CMB=None):
 
     # Call function
     if ((mass_split == 'normal') or
-            (mass_split == 'inverted') or (mass_split == 'equal')):
+            (mass_split == 'inverted') or (mass_split == 'equal') or
+            (mass_slit=='single'):
```

Comment:
This might read better as
```
if mass_split in ['normal', 'inverted', 'equal', 'single']:
```
Less lispy looking.  :)

Reply:
There is a syntax error here too. and flake8 will choke on the operator spaces potentially.

Reply:
Good call, will change this. Just found the syntax error also, sorry about that.